include/mysqli.php

```diff
@@ -115,7 +115,15 @@ function db_version() {
 }
 
 function db_timezone() {
-    return db_get_variable('system_time_zone', 'global');
+	$php_tz = ini_get('date.timezone');
+	$php_abbr = new DateTime('now', new DateTimeZone($php_tz));
+	$php_abbr = $php_abbr->format('T');
+	$sql_tz = db_get_variable('system_time_zone', 'global');
+	if ($php_abbr === $sql_tz)
+		return $php_tz;
+
+	echo '<i class="icon icon-warning-sign" title="Mismatch between \'date.timezone\' and \'mysql timezone\'"></i> ';
```

Comment:
mismatches are normal... you don't always have authority of the mysql server timezone. That echo might break some things... best to log in dashboard.